cli.js

```diff
@@ -283,6 +283,24 @@ Usage: $0 <command> [options]` // eslint-disable-line comma-dangle
   }, async () => {
     await loginHandler();
   })
+  .command('feedback <email> <message>', 'Provide your feedback on our product', (yargs0) => {
+    yargs0
+      .usage('Usage: $0 feedback <email> <message>')
+      .positional('email', {
+        describe: 'User email',
+        type: 'string',
+      })
+      .positional('message', {
+        describe: 'Feedback message',
+        type: 'string',
+      })
+      .example(
+        `  // Send feedback message to us with your email address
+          bn feedback "you@email.com" "Great Product!"`);
+  }, async (argv) => {
+    await handleCommand(argv, feedbackHandler);
+  })
+  // .strict()
```

Comment:
if this is intended (which I think it should, to allow spaces in the message, but can be in a future PR) a comment is better, such as `// notice: no .strict() on purpose`.
If it's not intended, let's keep it strict.